service/dap/daptest/client.go

```diff
@@ -89,6 +89,15 @@ func (c *Client) expectErrorResponse(t *testing.T, id int, message string) *dap.
 	return er
 }
 
+func (c *Client) ExpectInitializeResponseAndCapability(t *testing.T) *dap.InitializeResponse {
```

Comment:
ExpectInitializeResponseWithDefaultCapabilities?
And please check also add the other missing capabilities that we set now. The previous helper never got updated as things evloved :(

Reply:
Done. Would be better if it was a purely refactoring cl.